pkg/errors/armerrors.go

```diff
@@ -63,6 +63,10 @@ func RegionalQuotaHasBeenReached(err error) bool {
 	return azErr != nil && azErr.ErrorCode == OperationNotAllowed && strings.Contains(azErr.Error(), RegionalQuotaExceededTerm)
 }
 
+func LowPriorityQuotaHasBeenReached(err error) bool {
```

Comment:
can we comment on when we expect this, should we think of it as client/server, and what low priority actually means